changelog.d/2845.misc.rst

```diff
@@ -0,0 +1,3 @@
+Added :func:`bank_prune` to :module:`redbot.core.bank`
```

Comment:
As this function will be documented, this should not be a misc changelog. Only undocumented and internal changes should be labeled as misc.